pkg/resources/aws/aws.go

```diff
@@ -70,6 +72,8 @@ func ListResourcesAWS(cloud awsup.AWSCloud, clusterName string) (map[string]*res
 		ListVPCs,
 		// ELBs
 		ListELBs,
+		ListELBV2s,
```

Comment:
Does this delete the `listeners` too?

Reply:
Yes, delete LoadBalancer takes care of listeners as well.